packages/compat/src/v1-app.ts

```diff
@@ -215,6 +215,11 @@ export default class V1App implements V1Package {
       plugins.push([ModulesAPIPolyfill, { blacklist }]);
     }
 
+    // TODO: mega haxor
```

Comment:
as described above:

```js
   // this is reproducing what ember-cli-babel does. It would be nicer to just
    // call it, but it require()s all the plugins up front, so not serializable.	    // call it, but it require()s all the plugins up front, so not serializable.
    // In its case, it's mostly doing it to set basedir so that broccoli caching	    // In its case, it's mostly doing it to set basedir so that broccoli caching
    // will be happy, but that's irrelevant to us here.	    // will be happy, but that's irrelevant to us here.
```

The solution is to see if we can get ember-cli-babel to provide us a serializable variant of its plugins, that we nicely consume here. That would avoid the need for us to do this.